app/views/chugtypes/show.html.erb

```diff
@@ -37,7 +37,7 @@
   </aside>
 
   <div class='md:col-span-8'>
-    <% if @chugs_all_newest.count > 0 %>
+    <% if @newest.count > 0 %>
```

Comment:
Newest kun je nog anders noemen, nu is het niet obvious wat het is zonder de controller gezien te hebben.